tasks.py

```diff
@@ -194,3 +202,42 @@ def type_check(
         cmds.extend(["--follow-imports=normal"])
     # use pseudo-terminal for colorized output
     ctx.run(" ".join(cmds), echo=True, pty=True)
+
+
+@invoke.task(aliases=["get-stats"])
+def get_usage_stats_json(ctx):
+    """
+    Dump usage stats event examples to json file
+    """
+    if not is_ge_installed:
+        print(
+            "This invoke task requires Great Expecations to be installed in the environment. Please try again."
+        )
+        return
```

Comment:
```suggestion
    if not is_ge_installed:
       raise invoke.Exit(
            message="This invoke task requires Great Expecations to be installed in the environment. Please try again.",
            code=1
        )
```

Great job. The only suggestion I have would be to make this small, change.
This will cause the task to exit early just as with the return but it uses a `non-zero` exit code which indicates to the caller shell that the task did not succeed.
https://docs.pyinvoke.org/en/stable/api/exceptions.html#invoke.exceptions.Exit

You might also be able to remove the similar logic from `mv_usage_stats_json` as well since `get_usage_stats_json` will always be called first.
You may want to double-check that though.

But these are minor improvements. I'm approving even if you decide not to make this change.

Reply:
oo this is MUCH better. thank you @Kilo59